frontend/src/modules/history/components/Translation.js

```diff
@@ -251,172 +239,206 @@ export class TranslationBase extends React.Component<InternalProps, State> {
-                                        title='Unapprove'
-                                        name='unapprove'
-                                        onClick={ this.handleStatusChange }
-                                        disabled={ this.props.isActionDisabled }
-                                    />
-                                </Localized>
-                                :
-                                <Localized
-                                    id='history-Translation--button-approved'
-                                    attrs={{ title: true }}
-                                >
-                                    <button
-                                        className='unapprove fa'
-                                        title='Approved'
-                                        disabled
-                                    />
-                                </Localized>
+                            </div>
+                            <menu className='toolbar'>
 
-                            :
-                            // Approve Button
-                            ( canReview && !isReadOnlyEditor ) ?
-                                <Localized
-                                    id='history-Translation--button-approve'
-                                    attrs={{ title: true }}
-                                >
-                                    <button
-                                        className='approve fa'
-                                        title='Approve'
-                                        name='approve'
-                                        onClick={ this.handleStatusChange }
-                                        disabled={ this.props.isActionDisabled }
-                                    />
-                                </Localized>
-                                :
+                            { this.renderDiffToggle() }
+
+                            { (index === 0 || !canComment) ? null : <span className='divider'>&bull;</span> }
+
+                            { (!canComment && commentCount === 0) ? null :
                                 <Localized
-                                    id='history-Translation--button-not-approved'
+                                    id='history-Translation--button-comment'
                                     attrs={{ title: true }}
+                                    $commentCount={ commentCount }
                                 >
                                     <button
-                                        className='approve fa'
-                                        title='Not approved'
-                                        disabled
-                                    />
+                                        className={ commentCount === 0 ? 'toggle-comments' :
+                                            'toggle-comments'
+                                        }
```

Comment:
Since the className is now the same regardless of `commentCount` this should just set the className to `toggle-comments` without the ternary, correct?

Reply:
I went ahead and made this change. Just let me know if I'm missing something and the logic needs to be reapplied.

Reply:
That ternary was weird anyways. :) "If true X else X". :)

Reply:
Originally there were two different classes based on the condition, which made more sense 😄